llms/mlx_lm/models/llama.py

```diff
@@ -17,9 +17,12 @@ class ModelArgs(BaseModelArgs):
     rms_norm_eps: float
     vocab_size: int
     num_key_value_heads: int = None
+    attention_bias: bool = False
+    mlp_bias: bool = False
```

Comment:
Was there a reason you added these / changed this file?

Reply:
Granite 3B and 8B require them, and its different from the base llama models.

Reply:
Oh I see, I did not realize those models had `model_type = llama`. Makes sense.